src/tea_html.res

```diff
@@ -1,6 +1,6 @@
 open Vdom
 
-module Cmds = Tea_html_cmds
+//module Cmds = Tea_html_cmds
```

Comment:
If these are being removed, can you make sure to remove them, not just comment them out

Reply:
Yes, I just wanted you to see it before completely removing them .